sdk/core/core-amqp/samples/send.ts

```diff
@@ -1,26 +1,64 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to send messages/events to Service Bus/Event Hubs.
+*/
+
+import {
+  ConnectionContextBase,
+  CreateConnectionContextBaseParameters,
+  CbsResponse,
+  ConnectionConfig,
+  TokenType,
+  SharedKeyCredential
+} from "@azure/core-amqp";
 
-import * as dotenv from "dotenv";
-dotenv.config(); // Optional for loading environment configuration from a .env (config) file
 import { Sender, SenderOptions, EventContext, Message, Delivery } from "rhea-promise";
-import { authenticate, connectionContext, connectionConfig, path } from "./cbsAuth";
+
+// Define connection string and related entity path here
+const connectionString = "";
+const path = "";
+
+const connectionConfig = ConnectionConfig.create(connectionString, path);
+const parameters: CreateConnectionContextBaseParameters = {
+  config: connectionConfig,
+  connectionProperties: {
+    product: "MSJSClient",
+    userAgent: "/js-core-amqp",
+    version: "0.1.0"
+  }
+};
+const connectionContext = ConnectionContextBase.create(parameters);
+
+async function authenticate(audience: string): Promise<CbsResponse> {
+  await connectionContext.cbsSession.init();
+  const sharedTokenCredential = <SharedKeyCredential>connectionContext.tokenCredential;
```

Comment:
sharedTokenCredential -> sharedKeyCredential

Same for the receive sample

Reply:
Add a comment here that here, we use the shared key information in the connection string to perform the authentication. If they want to use Azure Active Directory, then refer to the sample where we show auth using aad.

Reply:
Updated!